docs/user/HTTP_TRAFFIC_SPLIT.md

```diff
@@ -0,0 +1,111 @@
+# HTTPRoute Traffic Split
+
+The [HTTPRoute][] resource allows one or more `backendRefs` to be provided. Requests will be routed to these upstreams if they match the rules of the HTTPRoute. If a backendRefs configured in an HTTPRoute is invalid, then HTTP responses will be returned with status code `500` for all of the requests that would have been sent to that backend.
```

Comment:
Instead of `backendRefs`, can we link to the upstream docs like [backendsRef](https://gateway-api.sigs.k8s.io/references/spec/#gateway.networking.k8s.io/v1beta1.BackendRef).

Reply:
For every reference, or just the first one?

Reply:
Just the first refernece.